Dockerfile

```diff
@@ -26,9 +26,6 @@ COPY --from=build-dummy /build /build
 FROM --platform=$BUILDPLATFORM ${GOLANG_IMAGE} AS base
 COPY --from=xx / /
 RUN echo 'Binary::apt::APT::Keep-Downloaded-Packages "true";' > /etc/apt/apt.conf.d/keep-cache
```

Comment:
Do we know if this is still needed by the way? I see it was added in https://github.com/moby/moby/commit/82c56755ea9e6c0ce197fbdfd41d0c5d1724b50c, and I was wondering "is the default different?" (maybe it is in the base-images)